Dockerfile

```diff
@@ -1,23 +1,27 @@
 FROM python:3.8
 
-ARG BUILD_ENV
-ARG CHROMEDRIVER_VERSION
+ARG BUILD_ENV=local
+ARG CHROMEDRIVER_VERSION=80.0.3987.106
 
 RUN mkdir -p /app/.apt/usr/bin
 
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
What's this for?

Reply:
This sets `bash` as the default shell for `RUN` instructions. Docker uses `sh` by default, which only checks the exit code of the final command. It also didn't recognize `pipefail` when I tried to set it, so I went with `bash` ([Docs](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#using-pipes)). 

I'm pretty sure that `apt-key` would have a non-zero exit code if there's a problem retrieving the key, but since we may add other RUN instructions containing pipes I think it makes sense to add it now.

Reply:
Got it. Can you add a comment in the Dockerfile with a link to those docs?

Reply:
Never mind, I did.